### PR TITLE
Add Korean translation

### PR DESCRIPTION
**Pull request by Johnmacrocraft**
- [x] This PR actually submits something - don't use this system as a messaging service!
- [x] This PR isn't duplicated - you can check if it is by scanning the small list - link is on the navigation bar for this repository.
- [x] This PR includes appropriate markdown for sections - e.g. code blocks for suggested code.
- [x] This PR description and comments in code is understandable - feel free to use your native language to write if you are not comfortable with English.
- [x] This PR has a single branch for itself in your fork - don't just commit to the master branch of your repository!
- [x] This PR has comments written in clear English - please don't write unreadable comments just for your eyes.

## Pull Request description
Just a translation submission from a (bored) translator, enjoy! :)

(Ignore branch name, I couldn't translate into Japanese because of laziness, and forgot to rename it. Probably I'll do it later if I have time)

### CODE REVIEW

**Comment by Johnmacrocraft** [comment]
It would be appreciated if you make strings for _enabled_ and _disabled_, instead of hardcoding them in the code, this makes translations weird for (I guess?) most languages

**Comment by Johnmacrocraft** [comment]
@Aericio The _pvp.success_ one.
- Q1: auto.*
- Q2: setowner.desc
- Q3: popup.available

**Comment by Johnmacrocraft** [comment]
@Aericio The _pvp.success_ one.
- Q1: auto.*
- Q2: setowner.desc
- Q3: popup.available

**Comment by Johnmacrocraft** [comment]
Some issues:
- Padding won't work properly because of Unicode characters, you'll probably want to fix that if possible
- /p list doesn't seem to work properly (on all languages)

**Comment by Johnmacrocraft** [comment]
Question:
- I'm assuming that the word _free_ in auto strings means something like _empty_, rather than _$0_, is that right?
- By _without charging anything_, does it mean that players can set owner without paying money? Just in case.
- Not sure which meaning is better for the word _open_, I definitely understood the string but still thinking of which one would be the best word to use. What's the exact meaning of that?

**Comment by Aericio** [comment]
> It would be appreciated if you make strings for enabled and disabled, instead of hardcoding them in the code, this makes translations weird for (I guess?) most languages

for when server starts/enables, the messages for them?

File: resources/eng.ini [hunk 1]
```diff
@@ -214,7 +214,7 @@ list.name=list
 list.alias=l
 list.desc=Shows all plots a player has
 list.usage=/p list [player: string]
-list.found=Plot {%0} at ({%1};{%2}),
+list.found="Plot {%0} at ({%1};{%2}), "
```
Comment (jasonwynn10) [inline resources/eng.ini]: Altering other language files is out of the scope of this PR and should not be included.

File: resources/cze.ini [hunk 2]
```diff
@@ -214,7 +214,7 @@ list.name=list
 list.alias=l
 list.desc=Zobrazi vsechny pozemky hrace
 list.usage=/p list [hrac]
-list.found=Pozemek {%0} na ({%1},{%2})
+list.found="Pozemek {%0} na ({%1},{%2})"
```
[the remark this answers is not in the document]
Comment (Johnmacrocraft) [reply resources/cze.ini]: /p list has never worked for me without quotation marks, try it

File: resources/eng.ini (hunk 1 above)
Comment (Johnmacrocraft) [reply resources/eng.ini]: Okay, I'm going to do that in the other PR

File: resources/cze.ini (hunk 2 above)
[the remark this answers is not in the document]
Comment (Johnmacrocraft) [reply resources/cze.ini]: On every language, except some of them that have added quotation marks

**Comment by Johnmacrocraft** [comment]
As I reverted my changes to other language files and the questions from the above have been resolved, you can merge this now.

**Pull request merged**